Exec/hydro_tests/acoustic_pulse/convergence_plm.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# echo the commands
+set -x
+
+DIM=3
+EXEC=./Castro${DIM}d.gnu.MPI.ex
+
+RUNPARAMS="castro.ppm_type=0 castro.time_integration_method=0"
+
+mpiexec -n 8 ${EXEC} inputs.3d.64 ${RUNPARAMS} amr.plot_file=acoustic_pulse_64_plm_plt &> 64.out
```

Comment:
similar to the convergence_ppm, maybe here should be should be inputs.64 instead of inputs.3d.64 ...etc.